src/data/sparse_page_source.h

```diff
@@ -338,28 +351,22 @@ class SparsePageSource : public DataSource<T> {
       if (page->data.Size() != 0) {
         writer.PushWrite(std::move(page));
       }
-
-      std::unique_ptr<dmlc::Stream> fo(
-          dmlc::Stream::Create(name_info.c_str(), "w"));
-      int tmagic = kMagic;
-      fo->Write(&tmagic, sizeof(tmagic));
-      info.SaveBinary(fo.get());
```

Comment:
Where did the meta info go?

Reply:
In the current flow, we create the row pages first and write the meta info there. This method now only creates column pages and sorted column pages from the existing row pages, so the meta info should be already written.